tests/integration/scripteditor.ts

```diff
@@ -91,7 +92,7 @@ describe('Script Editor tests', () => {
     cy.createNewScriptFile('Python');
 
     // Add some text to the editor
-    cy.get('span[role="presentation"]').type('print("test")');
+    cy.get('span[role="presentation"]').type('print("test")\n');
```

Comment:
To clarify, this is being added here as suggested by @ajbozarth to overcome a new code completer element that sometimes blocks the toolbar buttons during testing runtime.